src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -302,7 +306,7 @@ void EnBom_Update(Actor* thisx, GlobalContext* globalCtx) {
 
     Actor_SetHeight(&this->actor, 20.0f);
 
-    if (this->actor.params <= 0) {
+    if (this->actor.params <= BOMB_BODY) {
```

Comment:
I'm not sure using the enum value for 0 makes sense in checks like that. I would probably keep it as `<= 0`
And the same comment applies to bombf ofc